app/templates/footer.njk

```diff
@@ -49,7 +52,7 @@
             <a href="/$( _lang_ )$/media" class="nav-link text-muted">$( footer.media )$</a>
           </li>
           <li class="nav-item">
-            <a href="/$( _lang_ )$/tools/" class="nav-link text-muted">$( footer.tools-tools )$ <span class="badge badge-primary">New</span></a>
```

Comment:
What's this removing?

Reply:
The "new" badge, because the address and broadcast tools aren't really new anymore